shopelectro/management/commands/price.py

```diff
@@ -129,36 +158,63 @@ def filter_categories(utm):
         def prepare_products(categories_, utm):
             """Filter product list and patch it for rendering."""
             products = PriceFilter(utm).run(
-                Product.objects.active().filter_by_categories(categories_)
+                models.Product.objects.active().filter_by_categories(categories_)
             )
-            brands = Tag.objects.get_brands(products)
+            brands = models.Tag.objects.get_brands(products)
             return [
                 put_brand(put_crumbs(put_utm(product)), brands)
                 for product in products
             ]
 
         categories = (
-            filter_categories(utm) if utm != 'SE78'
-            else Category.objects.all()
+            filter_categories(self.target) if self.target != 'SE78'
+            else models.Category.objects.all()
         )
 
-        products = prepare_products(categories, utm)
+        products = prepare_products(categories, self.target)
 
         return {
             'base_url': settings.BASE_URL,
             'categories': categories,
             'products': products,
             'shop': settings.SHOP,
-            'utm': utm,
+            'utm': self.target,
         }
 
-    @classmethod
-    def generate_yml(cls, utm, file_name):
-        """Generate yml file."""
-        file_to_write = os.path.join(cls.BASE_DIR, file_name)
-        context = cls.get_context_for_yml(utm)
 
-        with open(file_to_write, 'w', encoding='utf-8') as file:
-            file.write(render_to_string('prices/price.yml', context).strip())
+class Category:
```

Comment:
What are these classes? How will they act? It would be nice to provide a comment for it to clarify what is going on.